### PR TITLE
feat: [asset-swapper] more hops via token adjacency

### PR DESCRIPTION
**Pull request by dekz**
## Description

Uses the `TokenAdjacencyGraph` for Uniswap/Sushi sells in both normal splitting and MultiHop Splitting.

[Sims](https://metabase.spaceship.0x.org/dashboard/139?run_id=practice-straight-reaction&adjusted_amount_bought_win_tolerance__bps_=0.5)

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Review by romainbutteaud — APPROVED** [review]
LGTM!
Just missing the CHANGELOG?

**Comment by dekz** [comment]
@romainbutteaud yeah I wait until the end to avoid publish merge conflicts 😂

**Pull request merged**
